service/step_defs_test.go

```diff
@@ -2730,7 +2730,7 @@ func (f *feature) deletionWorkerProcessesWhichResultsIn(volumeName, errormsg str
 	if volumeID == "" {
 		return fmt.Errorf("Could not find volumeID for volume %s", volumeName)
 	}
-	volumeName = DeletionPrefix + "csi-" + f.service.opts.ClusterPrefix + "-" + volumeName
```

Comment:
why this change is required?

Reply:
Since we are building the volume name here for the unit-tests, additional '-' to accommodate the namespace value